contracts/EpochTokenLocker.sol

```diff
@@ -205,12 +205,30 @@ contract EpochTokenLocker {
         balanceStates[user][token].balance = balanceStates[user][token].balance.add(amount);
     }
 
-    function subtractBalanceWithPendingWithdrawCheck(address user, address token, uint256 amount) internal {
+    /**
+     * The following function should be used to substract amounts from the current balances state.
+     * For the substraction, the current withdrawRequests are considered and effectively they are reducing
+     * the available balance.
+     */
+    function subtractBalance(address user, address token, uint256 amount) internal {
         require(amount <= getBalance(user, token), "Amount exceeds user's balance.");
-        subtractBalance(user, token, amount);
+        subtractBalanceUnchecked(user, token, amount);
     }
 
-    function subtractBalance(address user, address token, uint256 amount) internal {
+    /**
+     * The following function should be used to substract amounts from the current balance
```

Comment:
Subtract*

Reply:
Did this "substract" get merged? Maybe the auditors will catch it!

Reply:
I corrected it in the commit. No worriese